README.md

```diff
@@ -1,15 +1,16 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# rlistings <a href='https://github.com/insightsengineering/rlistings'><img src="man/figures/logo.png" align="right" height="200" width="200"/></a>
+# rlistings
 
 <!-- start badges -->
+
 [![Check
-🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://insightsengineering.github.io/rlistings/main/unit-test-report/)
```

Comment:
as suggested here, i am a bit inclined to keep the unit-test-report https://github.com/insightsengineering/rlistings/pull/202#discussion_r1513736161

if this is misleading, perhaps could do, 

unittest-report -- with the link to the report
check --  with the workflow check link

Reply:
@Melkiades , any thoughts?

Reply:
@shajoezhu I didn’t change anything in this part of the file so nothing should be missing unless the Rmd file was changed and not re-knit. I think the formatting just changed slightly when I knit the Rmd file, but I will double check and make changes if needed

Reply:
I see! It traces back to here. 

https://github.com/insightsengineering/rlistings/blame/d4e071dbb46f0652dbf425a599297d395d7e9ab8/README.Rmd#L19

@pawelru , should we keep using the unit-test-report link? or just switch it to the check history?

https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml is not very useful, is it?

Reply:
Hmmm... I don't have a strong view on this to be honest. For now, for consistency, I would suggest to go with what's in other packages and we can open a cross-repo discussion what should be there.